package.json

```diff
@@ -21,7 +21,6 @@
   ],
   "scripts": {
     "lint": "eslint *.js",
-    "pretest": "npm run lint",
```

Comment:
The `precommit` will still run both, but this aligns to our standard setup of individually explicit commands.